mmdet3d/models/backbones/pointnet2_sa_msg.py

```diff
@@ -102,15 +102,21 @@ def __init__(self,
                     cfg=sa_cfg,
                     bias=True))
             skip_channel_list.append(sa_out_channel)
-            self.aggregation_mlps.append(
-                ConvModule(
-                    sa_out_channel,
-                    aggregation_channels[sa_index],
-                    conv_cfg=dict(type='Conv1d'),
-                    norm_cfg=dict(type='BN1d'),
-                    kernel_size=1,
-                    bias=True))
-            sa_in_channel = aggregation_channels[sa_index]
+
+            cur_aggregation_channel = aggregation_channels[sa_index]
+            if cur_aggregation_channel is None:
```

Comment:
In the original implementation of PN++ for segmentation, we don't have this aggregation MLP. So I judge if it's None here.